imap/jmap_contact.c

```diff
@@ -1489,7 +1489,7 @@ static json_t *jmap_contact_from_vcard(struct vparse_card *card,
     hash_table labels = HASH_TABLE_INITIALIZER;
     construct_hash_table(&labels, 10, 0);
     for (entry = card->properties; entry; entry = entry->next) {
-        if (!strcasecmp(entry->name, "X-ABLabel")) {
+        if (!strcasecmp(entry->name, APPLE_LABEL_PROPERTY)) {
```

Comment:
Is entry->name guaranteed to be non-NULL? Other calls to `strcasecmpsafe` later in this commit suggest it isn't.

Reply:
I don't think any name will ever be NULL, but I'll use safe just to be safe